plugins/adminchat.sma

```diff
@@ -217,9 +217,9 @@ public cmdSayAdmin(id)
 	log_message("^"%s<%d><%s><>^" triggered ^"amx_chat^" (text ^"%s^")", name, userid, authid, message[1])
 	
 	if (is_user_admin(id)) // no diff here if admins have g_AdminChatFlag access or not, but we don't want to print "PLAYER"
-		format(message, charsmax(message), "(%L) %s :  %s", id, "ADMIN", name, message[1])
+		format(message, charsmax(message), "(%L) %s :  %s", LANG_PLAYER, "ADMIN", name, message[1])
```

Comment:
You can't do like that, you should re-format message for each player.

Reply:
I think you can either use `SetGlobalTransTarget` + `%l` or using `%s` + `fmt("%L", id, "ADMIN")`

Reply:
I read somewhere that this is fixed in 1.9, guess it's wrong. I changed it to use `%l`.

Reply:
@OciXCrom that is not how it works.

Reply:
@OciXCrom it is not how it works.

Reply:
Actually, I would cache `is_user_admin` and would do the check in the loop, and formatting should be made in `client_print`. It will be easier to translate `ADMIN` to the user's language.

Reply:
Done.